### PR TITLE
Add ERC: NFT Metadata Extension like JSON-LD

### PR DESCRIPTION
**Pull request by yoheinishikubo**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

**Comment by SamWilsn** [comment]
What is this exactly?

**Comment by yoheinishikubo** [comment]
@SamWilsn
Apologies for the delay.

I've been involved with projects that use NFTs as proof of event attendance and for carbon offsetting flights. 
These initiatives follow existing standards, some of which are documented on Schema.org. 

While there are numerous similar projects, they often employ varying data schemas, making it nearly impossible to integrate all the data seamlessly.

To address this, I propose adopting universal standards for NFT metadata that computers can easily utilize, akin to how HTML serves humans and JSON-LD serves computers.

**Comment by yoheinishikubo** [comment]
@SamWilsn 
Apologies for the confusion. It seems I made an error while migrating this PR from the original repository, resulting in it being empty. Could you please review the current status again?

File: ERCS/erc-7280.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+| schema | MUST             | object | The schema of the linked data. |
+| data   | MUST             | object | The data of the linked data.   |
+
+### Schema
+
+| name        | compliance level | type   | description                    |
+| ----------- | ---------------- | ------ | ------------------------------ |
+| uri         | MUST             | string | The URI of the schema.         |
+| name        | MUST             | string | The name of the schema.        |
+| description | OPTIONAL         | string | The description of the schema. |
```
Comment (SamWilsn) [inline ERCS/erc-7280.md:41]: It might be nice to use something like JSON Schema to specify these requirements. That's up to you though.

**Review by SamWilsn — APPROVED** [review]

**Pull request merged**